src/FamilyHubs.ServiceDirectory.Infrastructure/FamilyHubs.ServiceDirectory.Infrastructure.csproj

```diff
@@ -16,11 +16,12 @@
 	</PropertyGroup>
 
 	<ItemGroup>
-	  <PackageReference Include="FamilyHubs.ServiceDirectory.Shared" Version="1.1.15" />
+	  <PackageReference Include="FamilyHubs.ServiceDirectory.Shared" Version="1.1.20" />
 	  <PackageReference Include="FamilyHubs.SharedKernel" Version="1.0.4" />
+	  <PackageReference Include="Microsoft.AspNetCore.WebUtilities" Version="2.2.0" />
```

Comment:
`Microsoft.AspNetCore.WebUtilities` is a legacy package and can be problematic in later .net versions.

What functionality do you need from it?